providers/fips/fipsprov.c

```diff
@@ -342,22 +371,27 @@ static const OSSL_ALGORITHM fips_ciphers[] = {
     { "AES-128-CTR", "fips=yes", aes128ctr_functions },
     { "AES-256-XTS", "fips=yes", aes256xts_functions },
     { "AES-128-XTS", "fips=yes", aes128xts_functions },
-    /* TODO(3.0) Add aliases for these ciphers */
-    { "id-aes256-GCM", "fips=yes", aes256gcm_functions },
-    { "id-aes192-GCM", "fips=yes", aes192gcm_functions },
-    { "id-aes128-GCM", "fips=yes", aes128gcm_functions },
-    { "id-aes256-CCM", "fips=yes", aes256ccm_functions },
-    { "id-aes192-CCM", "fips=yes", aes192ccm_functions },
-    { "id-aes128-CCM", "fips=yes", aes128ccm_functions },
-    { "id-aes256-wrap", "fips=yes", aes256wrap_functions },
-    { "id-aes192-wrap", "fips=yes", aes192wrap_functions },
-    { "id-aes128-wrap", "fips=yes", aes128wrap_functions },
-    { "id-aes256-wrap-pad", "fips=yes", aes256wrappad_functions },
-    { "id-aes192-wrap-pad", "fips=yes", aes192wrappad_functions },
-    { "id-aes128-wrap-pad", "fips=yes", aes128wrappad_functions },
+    { "AES-256-GCM:id-aes256-GCM", "fips=yes", aes256gcm_functions },
+    { "AES-192-GCM:id-aes192-GCM", "fips=yes", aes192gcm_functions },
+    { "AES-128-GCM:id-aes128-GCM", "fips=yes", aes128gcm_functions },
+    { "AES-256-CCM:id-aes256-CCM", "fips=yes", aes256ccm_functions },
+    { "AES-192-CCM:id-aes192-CCM", "fips=yes", aes192ccm_functions },
+    { "AES-128-CCM:id-aes128-CCM", "fips=yes", aes128ccm_functions },
+    { "AES-256-WRAP:id-aes256-wrap:AES256-WRAP", "fips=yes",
+      aes256wrap_functions },
+    { "AES-192-WRAP:id-aes192-wrap:AES192-WRAP", "fips=yes",
+      aes192wrap_functions },
+    { "AES-128-WRAP:id-aes128-wrap:AES128-WRAP", "fips=yes",
+      aes128wrap_functions },
+    { "AES-256-WRAP-PADid-aes256-wrap-pad::AES256-WRAP-PAD", "fips=yes",
```

Comment:
Not sure what the double seperator before AES256-WRAP-PAD would do :)

Reply:
They are run-away sheep!  I've just nudged them back in their fold...

Reply:
@slontis prefers goats to sheep 🤣

Reply:
The best of both worlds
https://www.youtube.com/watch?v=N-lFK8iRG60